modules/auxiliary/gather/f5_bigip_cookie_disclosure.rb

```diff
@@ -32,8 +32,11 @@ def initialize(info = {})
 
     register_options(
       [
+        OptInt.new('RPORT', [true, 'The BigIP service port to listen on', 443]),
         OptString.new('TARGETURI', [true, 'The URI path to test', '/']),
-        OptInt.new('REQUESTS', [true, 'Number of requests to send to disclose back', 10])
+        OptBool.new('SSL', [true, 'Negotiate SSL/TLS for outgoing connections', true]),
```

Comment:
These already exist as advanced options. Are you doing this just to change the default?  If so, you can put them in the info hash above instead:
```
  'DefaultOptions' => {
    'SSL' => true,
    'SSLVersion' => 'TLS1',
  }
```